PowerEditor/src/WinControls/Preference/preferenceDlg.cpp

```diff
@@ -1133,7 +1213,22 @@ intptr_t CALLBACK DarkModeSubDlg::run_dlgProc(UINT message, WPARAM wParam, LPARA
 	{
 		case WM_INITDIALOG:
 		{
-			::SendDlgItemMessage(_hSelf, IDC_CHECK_DARKMODE_ENABLE, BM_SETCHECK, nppGUI._darkmode._isEnabled, 0);
+			// light mode : !_isEnabled

```

Comment:
use
```
case WM_INITDIALOG:
{
	const int topControlID = NppDarkMode::isWindowsModeEnabled() ? IDC_RADIO_DARKMODE_FOLLOWWINDOWS : NppDarkMode::isEnabled() ? IDC_RADIO_DARKMODE_DARKMODE : IDC_RADIO_DARKMODE_LIGHTMODE;
	::SendDlgItemMessage(_hSelf, topControlID, BM_SETCHECK, BST_CHECKED, 0);

	int id = IDC_RADIO_DARKMODE_BLACK;
```

all variables are already set so we don't need to use directly `nppGUI._darkmode.` variables